src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -67,17 +96,68 @@ export const BaseCard = forwardRef<CardHandle, BaseCardProps>(
       flip: (forceFaceUp?: boolean) => turnCard(forceFaceUp),
     }));
 
-    const onClick = () => !disableNativeEvents && turnCard();
+    const onClick = () => !disableNativeEvents && !isDragging && Date.now() - startDate < 200 && turnCard();
```

Comment:
Just for us to remember, it would be good to have a comment here why we needed to include a debounce

Reply:
Actually, is this the only way of handling this case?
I mean, `onClick` will, for sure, get triggered after `mouseDown`. 
Isn't just a matter of checking something like "is this OnClick triggered after a mouseMove or not"?

It seems like a hack having this kind of stuff

Reply:
I agree but how would you check that "is this OnClick triggered after a mouseMove or not"? The problem is that the onMouseDown is also triggered in that case and it sets the isDragged to true.

It is a hack indeed but could not find a way to make it work otherwise. If you find a better way I would be happy to change it 🙂

Reply:
Actually I just found a better way (less hacky let's say 😉), will push soon so let me know what you think